### PR TITLE
Add .git-blame-ignore-revs to make GitHub's blame view nicer

### PR DESCRIPTION
**Pull request by DHowett**
Commits mentioned in this file will be acknowledged by GitHub, but
skipped in the blame view. Other tools use this as well.

You can make git use it by passing
`--ignore-revs-file .git-blame-ignore-revs` to `git blame`.

The only commits we're ignoring right now are codebase-wide reformatting
or line endings changes.

### CODE REVIEW

**Comment by DHowett** [comment]
More info at https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view

**Review by zadjii-msft — APPROVED** [review]
this has a real petty energy to it, I love it

**Comment by DHowett** [comment]
@msftbot merge this in 1 minute

**Comment by Lo0oG** [comment]
How about including  12cdc5cd79321d71cc228d41775e03c8cc360fbe ?

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# To use it with "git", run
+# > git blame --ignore-revs-file ./.git-blame-ignore-revs
+
+# Reformatted the entire codebase
+9b92986b49bed8cc41fde4d6ef080921c41e6d9e
+
+# Line Endings changes
+cb7a76d96c92aa9fc7b03f69148fb0c75dff191d
+5bbf61af8c8f12e6c05d07a696bf7d411b330a67
+d07546a6fef73fa4e1fb1c2f01535843d1fcc212
```
Comment (DHowett) [inline .git-blame-ignore-revs:12]:
```suggestion
d07546a6fef73fa4e1fb1c2f01535843d1fcc212

# UTF-8 BOM changes
12cdc5cd79321d71cc228d41775e03c8cc360fbe
```

**Review by lhecker — APPROVED** [review]

File: .git-blame-ignore-revs [hunk 2]
```diff
@@ -10,3 +10,6 @@
 cb7a76d96c92aa9fc7b03f69148fb0c75dff191d
 5bbf61af8c8f12e6c05d07a696bf7d411b330a67
 d07546a6fef73fa4e1fb1c2f01535843d1fcc212
+
+# UTF-8 BOM changes
+12cdc5cd79321d71cc228d41775e03c8cc360fbe
```
Comment (lhecker) [inline .git-blame-ignore-revs:15]: Wait a second... Shouldn't this be https://github.com/microsoft/terminal/commit/ddae2a1d49d604487d3c963e5eacbeb73861d986?

Comment (DHowett) [reply .git-blame-ignore-revs:15]: how the heck did .. yes

File: .git-blame-ignore-revs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# Reformatted the entire codebase
+9b92986b49bed8cc41fde4d6ef080921c41e6d9e
+
+# Line Endings changes
+cb7a76d96c92aa9fc7b03f69148fb0c75dff191d
+5bbf61af8c8f12e6c05d07a696bf7d411b330a67
+d07546a6fef73fa4e1fb1c2f01535843d1fcc212
+
+# UTF-8 BOM changes
+12cdc5cd79321d71cc228d41775e03c8cc360fbe
```
Comment (DHowett) [inline .git-blame-ignore-revs:15]:
```suggestion
ddae2a1d49d604487d3c963e5eacbeb73861d986
```

**Pull request merged**